dns.config.yaml

```diff
@@ -0,0 +1,6 @@
+listening-port: 53
+remote-lookup-port: 42069
+thread-count: 1
+use-udp: true
+use-tcp: true
```

Comment:
The default should be false for now until #8

Reply:
ah gotcha good point